content/docs/concepts_features/app_prereqs.md

```diff
@@ -2,7 +2,6 @@
 title: "Application Requirements"
 description: "Application Requirements"
 type: docs
-aliases: ["/docs/application_requirements"]
```

Comment:
We should keep this alias because it exists because a page was moved.

Reply:
I'll add this back, though I don't think we should worry about preserving an old page reference given there are other fundamental things like URLs broken on the website that we should fix first.

Reply:
I agree and think you are right in general. We should keep this specific alias since it's the kind of thing we may end up having to add back, if we see it in 404 reports - it's not a bad thing for old links to redirect (and doesn't commit us to perfect redirection forever).